src/registry_schemas/schemas/order.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/order",
+    "type": "object",
+    "definitions": {},
+    "title": "Order Schema",
+    "required": [
+        "orderDetails"
+    ],
+    "properties": {
+        "fileNumber": {
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 20,
+            "description": "The court assigns each court order a unique file number up to 20 characters in length."
+        },
+        "orderDate": {
+            "type": "string",
+            "description": "The date and time of the order.",
+            "format": "date-time",
+            "examples": ["1970-01-01T00:00:00+00:00"]
+        },
+        "effectOfOrder": {
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 500,
+            "description": "planOfArrangement"
+        },
+        "orderDetails": {
```

Comment:
I believe that, for some filings, the UI calls this property, `details`.

1. https://github.com/bcgov/business-filings-ui/blob/c9afa338ea6b4c47d691a22b4b752cc59733710a/src/components/dialogs/AddStaffNotationDialog.vue#L337
2. https://github.com/bcgov/business-filings-ui/blob/c9afa338ea6b4c47d691a22b4b752cc59733710a/src/components/dialogs/AddStaffNotationDialog.vue#L350
3. https://github.com/bcgov/business-filings-ui/blob/c9afa338ea6b4c47d691a22b4b752cc59733710a/src/components/dialogs/AddStaffNotationDialog.vue#L380

Reply:
I didn't realize we have more places with these. Reverting this for now